media/css/hubs/_common.scss

```diff
@@ -13,23 +13,23 @@ $color-link-blue-dark: #175a77;
 // Section titles
 
 .section-title {
-    @include font-size(24px);
+    @include font-size-level4;
     margin: 0 0 20px;
 
     span {
         @include box-decoration-break(clone);
         background-color: #000;
         color: #fff;
         line-height: 1.3;
-        padding: 0 5px 0 60px;
+        padding: 0 5px 0 40px;
     }
 }
 
 /* -------------------------------------------------------------------------- */
 // CTA link
 
 a.cta-link {
-    @include font-size(18px);
+    @include font-size(14px);
```

Comment:
Hm, yeah. 16px Open Sans is about the same size as 18px Zilla Slab just because Zilla is so compact and unsuited for body text. So maybe this is a place to use `font-size(14px)` after all. But I'm inclined to just let this go as-is for now, we've fussed over this PR too much already.